src/convertToObject.js

```diff
@@ -12,7 +13,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const splitedString = sourceString.split(";");
+
+  const divide = splitedString.map((prop) => {
+    const dividedArr = prop.trim().split(":");
+
+    return dividedArr.map((prop) => prop.trim()).filter((prop) => prop !== "");
```

Comment:
![image](https://github.com/mate-academy/js_style-to-object/assets/59209337/db54068b-30c7-4a81-a754-40f398e648db)